plugins/null-ls/servers.nix

```diff
@@ -7,72 +7,39 @@
   helpers = import ./helpers.nix args;
   serverData = {
     code_actions = {
+      eslint = {package = pkgs.nodePackages.eslint;};
+      eslint_d = {package = pkgs.nodePackages.eslint_d;};
       gitsigns = {};
-      shellcheck = {
-        package = pkgs.shellcheck;
-      };
-      statix = {
-        package = pkgs.statix;
-      };
+      shellcheck = {package = pkgs.shellcheck;};
+      statix = {package = pkgs.statix;};
     };
     completion = {};
     diagnostics = {
-      flake8 = {
-        package = pkgs.python3Packages.flake8;
-      };
-      shellcheck = {
-        package = pkgs.shellcheck;
-      };
-      cppcheck = {
-        package = pkgs.cppcheck;
-      };
-      gitlint = {
-        package = pkgs.gitlint;
-      };
-      deadnix = {
-        package = pkgs.deadnix;
-      };
-      statix = {
-        package = pkgs.statix;
-      };
+      cppcheck = {package = pkgs.cppcheck;};
+      deadnix = {package = pkgs.deadnix;};
+      eslint = {package = pkgs.nodePackages.eslint;};
+      eslint_d = {package = pkgs.nodePackages.eslint_d;};
+      flake8 = {package = pkgs.python3Packages.flake8;};
+      gitlint = {package = pkgs.gitlint;};
+      shellcheck = {package = pkgs.shellcheck;};
+      statix = {package = pkgs.statix;};
     };
     formatting = {
-      phpcbf = {
-        package = pkgs.phpPackages.phpcbf;
-      };
-      alejandra = {
-        package = pkgs.alejandra;
-      };
-      nixfmt = {
-        package = pkgs.nixfmt;
-      };
-      nixpkgs_fmt = {
-        package = pkgs.nixpkgs-fmt;
-      };
-      prettier = {
-        package = pkgs.nodePackages.prettier;
-      };
-      black = {
-        package = pkgs.python3Packages.black;
-      };
-      fourmolu = {
-        package = pkgs.haskellPackages.fourmolu;
-      };
-      fnlfmt = {
-        package = pkgs.fnlfmt;
-      };
-      stylua = {
-        package = pkgs.stylua;
-      };
-      cbfmt = {
-        package = pkgs.cbfmt;
-      };
-      shfmt = {
-        package = pkgs.shfmt;
-      };
-      taplo = {
-        package = pkgs.taplo;
-      };
+      alejandra = {package = pkgs.alejandra;};
+      black = {package = pkgs.python3Packages.black;};
+      cbfmt = {package = pkgs.cbfmt;};
+      eslint = {package = pkgs.nodePackages.eslint;};
+      eslint_d = {package = pkgs.nodePackages.eslint_d;};
+      fnlfmt = {package = pkgs.fnlfmt;};
+      fourmolu = {package = pkgs.haskellPackages.fourmolu;};
+      nixfmt = {package = pkgs.nixfmt;};
+      nixpkgs_fmt = {package = pkgs.nixpkgs-fmt;};
+      phpcbf = {package = pkgs.phpPackages.phpcbf;};
+      prettier = {package = pkgs.nodePackages.prettier;};
+      prettier_d = {package = pkgs.nodePackages.prettier_d_slim;};
```

Comment:
I think that renaming all instances of `prettier_d` to `prettier_d_slim` would be a better option since a `prettierd` source already exists; doing so would avoid confusions.

Reply:
This makes sense. I also noticed that null-ls has both `prettierd` and `prettier_d_slim` sources, so I may as well make this change.